source/documentation/breaking-changes/mixed-decls.md

```diff
@@ -0,0 +1,118 @@
+---
+title: 'Breaking Change: Mixed Declarations'
+introduction: >
+  CSS is changing the way it handles declarations mixed with nested rules, and
+  we want to make sure Sass matches its behavior.
+---
+
+## The Story So Far
+
+Historically, if you mixed together nested rules and declarations in Sass, it
+would pull all the declarations to the beginning of the rule to avoid
+duplicating the outer selector more than necessary. For example:
+
+{% codeExample 'mixed-declarations-old' %}
+  .example {
+    color: red;
+
+    a {
+      font-weight: bold;
+    }
+
+    font-weight: normal;
+  }
+  ===
+  .example
+    color: red
+
+    a
+      font-weight: bold
+
+
+    font-weight: normal
+  ===
+  .example {
+    color: red;
+    font-weight: normal;
+  }
+
+  .example a {
+    font-weight: bold;
+  }
+{% endcodeExample %}
+
+When [plain CSS Nesting] was first introduced, it behaved the same way. However,
+after some consideration, the CSS working group decided it made more sense to
```

Comment:
Could you add a CSSWG issue that would be helpful for authors to understand why the CSS syntax does what it does?

I think it should be: https://github.com/w3c/csswg-drafts/issues/8738

Reply:
Good idea. I've sent out https://github.com/sass/sass-site/pull/1113